geolocator_android/pubspec.yaml

```
@@ -18,7 +18,7 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  geolocator_platform_interface: ^3.0.0+1
+  geolocator_platform_interface: ^4.0.0+1
```

Version `4.0.0+1` doesn't exists, please update to `4.0.3`.
```suggestion
  geolocator_platform_interface: ^4.0.3
```